content/00.front-matter.md

```diff
@@ -0,0 +1,12 @@
+# Manubot Rootstock: Manuscript Title
+
+## Authors
+
+{% for author in authors %}
++ **{{author.full_name}}**<br>
+  [\@{{author.github_username}}](https://github.com/{{author.github_username}}) ·
+  [{{author.orcid}}](https://orcid.org/{{author.orcid}}) ·
+  {{author.affiliations}} ·
+  Funded by {{author.funders}}
```

Comment:
That might work well.  We can merge this and defer that decision (cc @evancofer).